docs/azure/sdk/authentication/create-token-credentials-from-configuration.md

```diff
@@ -0,0 +1,151 @@
+
+The following credential types are supported via configuration:
+
+* [ClientCertificateCredential](#create-a-clientcertificatecredential-type)
+* [ClientSecretCredential](#create-a-clientsecretcredential-type)
+* [DefaultAzureCredential](#create-a-defaultazurecredential-type)
+* [ManagedIdentityCredential](#create-a-managedidentitycredential-type)
+* [WorkloadIdentityCredential](#create-a-workloadidentitycredential-type)
+
+## Configure Azure credentials
+
+Azure service clients registered with the <xref:Microsoft.Extensions.Azure.AzureClientServiceCollectionExtensions.AddAzureClients%2A> method are automatically configured with an instance of `DefaultAzureCredential` if no explicit credential is supplied via the <xref:Microsoft.Extensions.Azure.AzureClientBuilderExtensions.WithCredential%2A> extension method. You can also override the global `DefaultAzureCredential` using credential values from configuration files when registering a client to create a specific credential type:
+
+```csharp
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    // Register BlobServiceClient using credentials from appsettings.json
+    clientBuilder.AddBlobServiceClient(builder.Configuration.GetSection("Storage"));
+
+    // Register ServiceBusClient using the fallback DefaultAzureCredential credentials
+    clientBuilder.AddServiceBusClientWithNamespace(
+        "<your_namespace>.servicebus.windows.net");
+});
+```
+
+The associated _appsettings.json_ file:
+
+```json
+"Storage": {
+    "serviceUri": "<service_uri>",
+    "credential": "managedidentity",
+    "clientId":  "<clientId>"
+}
+```
+
+The following credential types also support the `AdditionallyAllowedTenants` property, which specifies additional Microsoft Entra tenants beyond the default tenant for which the credential may acquire tokens:
+
+* [ClientCertificateCredential](#create-a-clientcertificatecredential-type)
+* [ClientSecretCredential](#create-a-clientsecretcredential-type)
+* [DefaultAzureCredential](#create-a-defaultazurecredential-type)
+
+Add the wildcard value "*" to allow the credential to acquire tokens for any Microsoft Entra tenant the logged in account can access. If no tenant IDs are specified, this option will have no effect on that authentication method, and the credential will acquire tokens for any requested tenant when using that method.
+
+```json
+{
+    "additionallyAllowedTenants":  "<tenant-ids-separated-by-semicolon>"
+}
+```
+
+### Create a `ManagedIdentityCredential` type
+
+You can create both user-assigned and system-assigned managed identities using configuration values. Add the following configuration values to your _appsettings.json_ file to create an instance of <xref:Azure.Identity.ManagedIdentityCredential?displayProperty=fullName>.
+
+#### User-assigned managed identities
+
+1. Specify a user-assigned managed identity via a client ID:
```

Comment:
I think this should be a bulleted list instead. We should also clarify that you choose one approach or the other.